com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -1889,6 +1945,133 @@ private void InjectWriteAndCallBlocks(MethodDefinition methodDefinition, CustomA
             instructions.ForEach(instruction => processor.Body.Instructions.Insert(0, instruction));
         }
 
+        private void GenerateVariableInitialization(TypeDefinition type)
+        {
+            foreach (var methodDefinition in type.Methods)
+            {
+                if (methodDefinition.Name == k_NetworkBehaviour___initializeVariables)
+                {
+                    // If this hits, we've already generated the method for this class
+                    // because a child class got processed first.
```

Comment:
```suggestion
                    // If this hits, we've already generated the method for this class because a child class got processed first.
```

^